include/identity-based/IdentityBased.h

```diff
@@ -0,0 +1,12 @@
+#ifdef __CRYPTID_IDENTITY_BASED
+
+#ifndef __CRYPTID_IDENTITY_BASED_H
+#define __CRYPTID_IDENTITY_BASED_H
```

Comment:
Please make sure, that header files start with an include guard because of the following reasons:
  - generally, that's what's expected,
  - underdocs assumes this layout,
  - This makes it harder to misuse the library.  If the include guard is the first condition, then the feature guard is evaluated only once. Thus, there can be no scenario where the client code messes up the defines and unexpected things occur. Include guards are safer, because we don't even mention them in the docs as opposed to the feature guards.

Reply:
Fair reasons. I will do the modifications.